### PR TITLE
Application is functional

### PR DESCRIPTION
**Pull request by haylau**

### CODE REVIEW

**Comment by mohazahid** [comment]
cause of me

**Review by mohazahid — APPROVED** [review]

**Review by mohazahid — APPROVED** [review]

**Review by mohazahid — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+async function queryWord (word: string): Promise<string> {
+  const response = await fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${word}`)
+  const data = await response.json()
+  console.log(data[0].meanings[0].definitions[0].definition)
+  return data[0].meanings[0].definitions[0].definition
+}
+let word
+do {
+  word = wl.getWord()
```
Comment (akumar282) [inline src/main.ts:23]: @mohazahid your so much better then me

File: src/wordList.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { fileURLToPath } from 'url'
+
+const DEFAULTFILEPATH = path.resolve(path.dirname(fileURLToPath(import.meta.url)), '../engmix.txt')
+
+// const DEFAULTFILEPATH = '../engmix.txt' // potentially will change
+
```
Comment (akumar282) [inline src/wordList.ts]: I love you @haydenlauritzen

**Review by akumar282 — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (haylau) [reply src/main.ts:23]: ~fuck you~ thank you abhi for teaching me typescript

Comment (mohazahid) [reply src/main.ts:23]: shut up abhi

File: src/wordList.ts (hunk 2 above)
Comment (haylau) [reply src/wordList.ts]: fucking yes abhi is amazing

Comment (mohazahid) [reply src/wordList.ts:8]: What a loser

**Review by akumar282 — APPROVED** [review]

**Comment by haylau** [comment]
fuck all of you

**Pull request merged**